packages/api/db/migration/20210812140444_cleaning_task.js

```diff
@@ -0,0 +1,27 @@
+exports.up = function (knex) {
+  return Promise.all([
+    knex('task_type').insert({
+      task_name: 'Cleaning',
+      task_translation_key: 'CLEANING',
+    }),
+    knex('task_type').where({ task_name:'AUG_20201_MIGRATION_PLACEHOLDER' }).del(),
+    knex.raw('ALTER TABLE task ALTER COLUMN type DROP DEFAULT'),
```

Comment:
Having the placeholder there was a mistake. The mapping between task and task_type was done already when the migration from logs occured, it was a "just in case a log has no task_type". But that did not end up happening